lib/gui/app/pages/main/controllers/image-selection.js

```diff
@@ -156,7 +157,7 @@ module.exports = function (
   this.openImageSelector = () => {
     analytics.logEvent('Open image selector')
 
-    if (process.env.ETCHER_EXPERIMENTAL_FILE_PICKER) {
+    if (settings.get('experimentalFilePicker')) {
```

Comment:
I wonder if this would be a good time to rename this setting something other than `experimentalFilePicker`? i.e. before it starts getting written into actual config files? (after which time it might become harder to rename it?)
IYSWIM.